src/main/java/ch/njol/skript/bukkitutil/BukkitUnsafe.java

```diff
@@ -57,6 +57,9 @@ public class BukkitUnsafe {
 		return getMaterialFromNamespacedId(id);
 	}
 
+	@Nullable
+	public static Material getMaterialFromMinecraftId(String id) {
```

Comment:
```suggestion
	public static @Nullable Material getMaterialFromMinecraftName(String name) {
```
current version implies that it uses numeric ids

Reply:
I dont think i tried to edit this at all. probably just messed up some merging or something